src/components/PostsList/PostsList.tsx

```diff
@@ -1,37 +1,66 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import './PostsList.scss';
 
-export const PostsList: React.FC = () => (
-  <div className="PostsList">
-    <h2>Posts:</h2>
-
-    <ul className="PostsList__list">
-      <li className="PostsList__item">
-        <div>
-          <b>[User #1]: </b>
-          sunt aut facere repellat provident occaecati excepturi optio
-        </div>
-        <button
-          type="button"
-          className="PostsList__button button"
-        >
-          Close
-        </button>
-      </li>
-
-      <li className="PostsList__item">
-        <div>
-          <b>[User #2]: </b>
-          et ea vero quia laudantium autem
-        </div>
-
-        <button
-          type="button"
-          className="PostsList__button button"
-        >
-          Open
-        </button>
-      </li>
-    </ul>
-  </div>
-);
+import { loadUsersPosts } from '../../api/posts';
+
+type Props = {
+  handleOpenPostDetails: (postId: number) => void;
+  postId: number;
+  selectorValue: number;
+};
+
+export const PostsList: React.FC<Props> = ({
+  handleOpenPostDetails,
+  postId,
+  selectorValue,
+}) => {
+  const [postList, setPostList] = useState<Post[]>([]);
+
+  const getUserPosts = async () => {
+    const userPostsFromServer = await loadUsersPosts();
+
+    if (selectorValue) {
+      return setPostList(userPostsFromServer.filter(post => post.userId === selectorValue));
```

Comment:
use this ability of API 
<img width="565" alt="image" src="https://user-images.githubusercontent.com/64437769/154463112-af3e82ae-e1c8-4f90-a993-bc1268fbeea2.png">